prime-router/src/main/kotlin/azure/SettingsFacade.kt

```diff
@@ -326,6 +326,10 @@ class SenderAPI
     topic: String,
     customerStatus: CustomerStatus = CustomerStatus.INACTIVE,
     schemaName: String,
+    keys: List<JwkSet>? = null,
```

Comment:
If it has a default value, does it make sense that it's also optional? It might, but having both has always confused me.

What does it mean to be optional but ALSO have a default value if it's left off?

Reply:
My understanding is that in Kotlin you need to mark it as nullable for it to accept null as a value at all.